api-server/server/boot/donate.js

```diff
@@ -246,24 +258,89 @@ export default function donateBoot(app, done) {
 
   const pubKey = keys.stripe.public;
   const secKey = keys.stripe.secret;
+  const paypalKey = keys.paypal.client;
+  const paypalSec = keys.paypal.secret;
   const hmacKey = keys.servicebot.hmacKey;
   const secretInvalid = !secKey || secKey === 'sk_from_stripe_dashboard';
   const publicInvalid = !pubKey || pubKey === 'pk_from_stripe_dashboard';
+
+  const paypalSecretInvalid =
+    !paypalKey || paypalKey === 'id_from_paypal_dashboard';
+  const paypalPublicInvalid =
+    !paypalSec || paypalSec === 'secret_from_paypal_dashboard';
   const hmacKeyInvalid =
     !hmacKey || hmacKey === 'secret_key_from_servicebot_dashboard';
+  const paypalInvalid = paypalPublicInvalid || paypalSecretInvalid;
+  const stripeInvalid = secretInvalid || publicInvalid || hmacKeyInvalid;
```

Comment:
These should be explicitly renamed to refer to stripe. Also since the HMAC key comes from servicebot, a separate service we should be treating it as such.